modules/role/manifests/st2.pp

```diff
@@ -22,4 +20,12 @@
       include ::profile::hubot
     }
   }
+
+  if $_enterprise_token {
+    include ::profile::st2flow
+    include ::profile::st2rbac
+    include ::profile::enterprise_auth_backend_ldap
+  } else {
+    include ::profile::auth_backend_pam
```

Comment:
I do need to add that I do agree that we need to add support for selecting LDAP backend to installer and everything which goes with that (this is probably a slightly bigger change though).